docs/examples-rust-wasi.md

```diff
@@ -1,101 +1,61 @@
-# WASI
+# WASI (Preview 2)
```

Comment:
Nowadays we try to call this "WASIp2", so mind renaming that here?

Reply:
done